### DISCUSSION TITLE
EV3RTでのBluetoothによるファイル転送ができない件について教えてください

### DISCUSSION THREAD

**Discussion by kitamihc** [toppers/users-forum#37] [ET-Robocon]
初歩的な質問です。
EV3にBluetoothで接続してファイル転送をしようとしたのですが、うまくいきません。
やり方が間違っているのかもしれません。
教えていただけないでしょうか。

PC: Windows10 (1803)
Bluetooth : BUFFALO BSBT4D200
Generic Bluetooth Radio
EV3RT Version 1.1

![ev3bt02](https://user-images.githubusercontent.com/88178983/127605825-046d2348-eff9-4d39-af97-33630883817d.png)
![ev3bt01](https://user-images.githubusercontent.com/88178983/127605842-6c911185-87f0-4db2-9432-5ab12c365384.png)

EV3の画面に出ているipと一致しています。

**Comment by ykominami** [comment]
@kitamihc  コメントが遅れすみません。ご質問ですが、EV3RTは、ETロボコン向けのTOPPERS成果物であるため、カテゴリは「Q&A」ではなく[ET-Robocon](https://github.com/toppers/users/discussions/categories/et-robocon)にお願いします。先ほど、users-forumの説明にも、EV3RTについて記述を追加しました。
github.comのDiscussion機能には、discussionをカテゴリ間で移動させる機能がないため、改めて[ET-Robocon]にnew discussion して、同一内容を書き込んでもらえませんでしょうか。[ET-Robocon]にdiscussionが追加された後、こちらはクローズとさせてください。

**Reply by ykominami** [reply]
@kitamihc 私が勘違いしていました。カテゴリはmaintainer権限で変更可能でした。現在はET-roboconになっていますので、このままで構いません。すみませんでした。
ラベル"question"を付加しました。
質問に回答される方は、（私のコメントにではなく）直接質問にAnswerをしてくださいl

**Comment by namerikawatym** [comment]
①EV3側で「App Loader」から「Bluetooth PAN」を選択。
②PC側で「Mindstorms EV3」とBluetooth PANに接続。
③
`$ make upload`
コマンドで転送

でしょうか。
